roles/openshift_node/templates/node.service.j2

```diff
@@ -6,18 +6,12 @@ After=ovsdb-server.service
 After=ovs-vswitchd.service
 Wants={{ openshift.docker.service_name }}.service
 Documentation=https://github.com/openshift/origin
-Requires=dnsmasq.service
```

Comment:
Probably still needs to start after dnsmasq.

Reply:
This I moved into a drop-in that's in openshift_node_dnsmasq role. I'll update the PR description to hilight that change.